c2corg_ui/static/js/documentservice.js

```diff
@@ -0,0 +1,51 @@
+goog.provide('app.Document');
+
+goog.require('app');
+
+
+/**
+ * This services is used to acces the document properties
+ * within other controllers, for example to set the properties in one 
+ * controller and then read them in another.
+ * @constructor
+ */
+app.Document = function() {
```

Comment:
Is this service name explicit enough?
Will it be used for something else than checking associations?